src/app.js

```diff
@@ -2,48 +2,36 @@ const { app } = require("./support/setupExpress");
 const { query } = require("./support/db");
 const { gameOfThronesEpisodes } = require("./data/gameOfThronesData");
 
-/** 
- @typedef {import('./data/episodeType').Episode} Episode
-*/
 
-//You can delete this once you see the episodes have loaded ok.
-summariseEpisodesToConsole(gameOfThronesEpisodes);
 
-//configure the server's route handlers
 app.get("/", (req, res) => {
-    res.render("pages/index");
+    res.render("pages/index", { 
+        
+    });
 });
 
-app.get("/db-test", async (req, res) => {
-    try {
-        const dbResult = await query("select now()");
-        const rows = dbResult.rows;
-        res.json(rows);
-    } catch (err) {
-        console.error(err);
-        res.status(500).send(
-            "Sorry, an error occurred on the server.  Ask the dev team to check the server logs at time " +
-                new Date(),
-        );
-    }
+app.get("/fullepisodes", (req, res) => { //cannot have spaces between nav names
+    res.render("pages/fullEpisodes", {
+        episodeArray: gameOfThronesEpisodes  
+    });
 });
 
-/**
- * You can delete this function.  It demonstrates the use of the Episode type in JSDoc.
- * @param {Episode[]} episodes
- * @returns void
- */
-function summariseEpisodesToConsole(episodes) {
-    console.log(`Loaded ${episodes.length} episodes`);
-    console.log("The first episode has name of " + episodes[0].name);
+function makeEpisodeCode(episode) {
+    const padSeason = String(episode.season).padStart(2, "0");
+    const padEpisode = String(episode.number).padStart(2, "0");
+
+    return `S${padSeason}E${padEpisode}`;
 }
+//make this function accessable in any ejs in our app.
+app.locals.makeEpisodeCode=makeEpisodeCode;
+
+//level 150
```

Comment:
This comment doesn't belong at all.  it doesn't help anyone to understand this part of the code.  If you want to leave a statement of what level you've got to, put it in the readme.md, not half way through one specific project file.